### DISCUSSION TITLE
Accessing Web Interface

### DISCUSSION THREAD

**Discussion by lconne81** [Security-Onion-Solutions/securityonion#3332] [Unsupported Versions]
I've opted to access the Security Onion Console via hostname. But when I tried to type the hostname into my browser, I get this message as you see below. This was done using Security Onion version 2.3.30. How do I resolve this?
![SO Web Interface Error](https://user-images.githubusercontent.com/80169050/110215321-23194a00-7e77-11eb-990c-f8a0cfb53191.JPG)

**Comment by TOoSmOotH** [comment]
If you chose hostname then you need to be able to resolve the name. Looks like this doesn't resolve for you so either add it to DNS or your hosts file.

http://www.steves-internet-guide.com/hosts-file/

**Reply by lconne81** [reply]
How do I do that exactly? The site you showed me explains how host names work, but I'm still not sure how to get mine's to work for me in this situation.

**Reply by TOoSmOotH** [reply]
The article tells you how to modify your hosts file about 3/4 the way down. This has nothing to do with security onion.

**Reply by lconne81** [reply]
So I should basically map my host name to the IP address I selected when I installed SO?

**Reply by dougburks** [reply]
@nokcy From the Discussions guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
